R/deps.R

```diff
@@ -529,6 +529,14 @@ remote_deps <- function(pkg, ...) {
   res
 }
 
+upgrade_default <- function() {
+
+  env <- Sys.getenv("R_REMOTES_UPGRADE", unset = NA)
```

Comment:
Couldn't this whole function just be

```suggestion
  Sys.getenv("R_REMOTES_UPGRADE", "ask")
```